src/containers/signup/index.tsx

```diff
@@ -19,12 +19,19 @@ const { Title, Paragraph } = Typography;
 
 type SignupProps = UserAuthenticationReducerState;
 
+interface SignupFormData extends SignupRequest {
+  confirmPassword: string;
+}
+
 const Signup: React.FC<SignupProps> = ({ tokens }) => {
   const dispatch = useDispatch();
   const history = useHistory();
 
-  const onFinish = (values: SignupRequest) => {
-    dispatch(signup(values));
+  const onFinish = (values: SignupFormData) => {
+    // confirmPassword is checked in the form but should not be sent in the request
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { confirmPassword, ...request } = values;
+    dispatch(signup(request));
```

Comment:
interesting... okay

Reply:
Yeah pretty weird. We could change the backend to not fail with extra parameters in the request but having it there will probably save us from bugs in the long term. If you have any cleaner way of getting rid of unwanted parameters in a JS object then we can do that here instead of the destructing call